src/components/Match/matchColumns.jsx

```diff
@@ -428,7 +436,20 @@ export const supportColumns = [
 ];
 
 export const chatColumns = [
-  heroTdColumn, {
+  {
+    displayName: strings.filter_is_radiant,
+    field: 'team',
```

Comment:
idk if team is a valid property here.  We don't use it anyway, so maybe should leave it blank